integration_tests/e2e/dpsFrontEndComponents.cy.ts

```diff
@@ -0,0 +1,306 @@
+import DashboardInitialPage from '../pages/dashboard/initial'
+import Page from '../pages/page'
+import { SUPERVISOR_USER } from '../factory/user'
+import { cleanString } from '../support/utilities'
+
+describe.skip('DPS Front End Components', () => {
```

Comment:
deliberately skipped as this will not work until we migrate

covered in https://dsdmoj.atlassian.net/browse/SI-409